db/seeds.rb

```diff
@@ -5,3 +5,51 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+Enneagram.destroy_all
+MyersBrigg.destroy_all
+Friendship.destroy_all
+User.destroy_all
+
+e1 = Enneagram.create(number: 1, name: 'The Reformer', description: 'Ones are conscientious and ethical, with a strong sense of right and wrong… Well-organized, orderly, and fastidious, they try to maintain high standards, but can slip into being critical and perfectionistic.')
+e2 = Enneagram.create(number: 2, name: 'The Helper', description: 'Twos are empathetic, sincere, and warm-hearted... They are well-meaning and driven to be close to others, but can slip into doing things for others in order to be needed.')
+e3 = Enneagram.create(number: 3, name: 'The Achiever', description: 'Threes are self-assured, attractive, and charming... They are diplomatic and poised, but can also be overly concerned with their image and what others think of them.')
+e4 = Enneagram.create(number: 4, name: 'The Individualist', description: 'Fours are self-aware, sensitive, and reserved. They are emotionally honest, creative, and personal, but can also be moody and self-conscious.')
+e5 = Enneagram.create(number: 5, name: 'The Investigator', description: 'Fives are alert, insightful, and curious... Independent, innovative, and inventive, they can also become preoccupied with their thoughts and imaginary constructs.')
+e6 = Enneagram.create(number: 6, name: 'The Loyalist', description: 'Sixes are reliable, hard-working, responsible, and trustworthy. Excellent troubleshooters, they foresee problems and foster cooperation, but can also become defensive, evasive, and anxious.')
+e7 = Enneagram.create(number: 7, name: 'The Enthusiast', description: 'Sevens are extroverted, optimistic, versatile, and spontaneous. Playful, high-spirited, and practical, they can also misapply their many talents, becoming over-extended, scattered, and undisciplined.')
+e8 = Enneagram.create(number: 8, name: 'The Challenger', description: 'Eights are self-confident, strong, and assertive. Protective, resourceful, straight-talking, and decisive, but can also be ego-centric and domineering.')
+e9 = Enneagram.create(number: 9, name: 'The Peacemaker', description: 'Nines are accepting, trusting, and stable. They are usually creative, optimistic, and supportive, but can also be too willing to go along with others to keep the peace.')
+
+mb1 = MyersBrigg.create(type: 'ISTJ', name: 'The Logistician', description: 'These people tend to be reserved yet willful, with a rational outlook on life. They compose their actions carefully and carry them out with methodical purpose.')
```

Comment:
Also perfect. I'd love to learn more about the how 'e6', 'mb6', etc are use in the back-end.